.github/workflows/monorepo-wiki.yml

```diff
@@ -0,0 +1,80 @@
+name: 'Monorepo: Wiki'
+on:
+  repository_dispatch:
+    types: [monorepo_release]
+
+jobs:
+  update-wiki:
+    runs-on: ubuntu-latest
+    name: 'Update Wiki'
+    steps:
+
+      - uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: unused
+          run_id: ${{ github.event.client_payload.run_id }}
+          path: outputs
+
+      - name: Checkout wiki
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository }}.wiki
+          token: ${{ secrets.GH_BOT_PERSONAL_ACCESS_TOKEN }}
+          path: wiki
+          fetch-depth: 0
+      - name: Ensure wiki folder
+        run: "[ -d wiki/generated ] || mkdir wiki/generated"
+      - name: Display structure of retrieved files
+        run: ls -R
+
+      - uses: actions/github-script@v2
+        id: write-changes
+        with:
+          script: |
+            const { owner, repo } = context.repo;
+            const { readdir, readFile, writeFile } = require('fs').promises;
+            const utf8 = { encoding: 'utf-8' };
+            const msgParts = [];
+
+            for (const folder of await readdir('outputs', { withFileTypes: true })) {
+              if (!folder.isDirectory()) continue;
+              const readText = (name) => readFile(name, utf8).then(x => x.trim());
+
+              const wikiPath = `wiki/generated/${folder.name}.md`;
+              const prevWiki = await readText(wikiPath)
+                .catch(err => err.code === 'ENOENT' ? '' : Promise.reject(err));
```

Comment:
The empty string here should be replaced with a constant including minimal instructions, such as `<!-- Add your introduction or context documentation here -->`